railties/lib/rails/railtie/forker.rb

```diff
@@ -0,0 +1,49 @@
+module Rails
+  class Forker
+    autoload :Runner, 'rails/railtie/forker/runner'
+
+
+    def runner=(runner)
+      @runner=runner 
+    end
+
+    def options
+      @options ||= {}
+    end
+
+    def options=(o)
+      options.merge!(o)
+    end
+
+    def before_fork(&blk)
```

Comment:
No problem :) -- master's compiled RDoc documentation can be found [here](http://edgeapi.rubyonrails.org) -- `ActiveSupport::Callbacks`' documentation is available [here](http://edgeapi.rubyonrails.org/classes/ActiveSupport/Callbacks.html).